Fix incorrect delay when setting WS2812 (and similar) leds (#9302)
qmk/qmk_firmware: docs/ws2812_driver.md

## Driver configuration

### All drivers

> Different versions of the addressable LEDs have differing requirements for the RESET period between frames. 

Reviewer: ```suggestion
Different versions of the addressable LEDs have differing requirements for the T<sub>RST</sub> period between frames.
```

> The default setting is 280 microseconds, which should work for most cases, but this can be overridden in your config.h. e.g.:

Reviewer: ```suggestion
The default setting is 280 µs, which should work for most cases, but this can be overridden in your config.h. e.g.:
```

```c

> #define WS2812_DELAY_MICROSECONDS 80

Reviewer: ```suggestion
#define WS2812_TRST_US 80
```
